src/test/rbd_mirror/image_replayer/test_mock_CreateImageRequest.cc

```diff
@@ -293,13 +288,14 @@ class TestMockImageReplayerCreateImageRequest : public TestMockFixture {
         })));
   }
 
-  void expect_clone_image(CreateCloneImage &create_clone_image,
-                          const std::string &local_image_name,
-                          const std::string &global_image_id,
-                          const std::string &remote_mirror_uuid, int r) {
-    EXPECT_CALL(create_clone_image, clone(local_image_name, global_image_id,
-                                          remote_mirror_uuid))
-      .WillOnce(Return(r));
+  void expect_clone_image(MockCloneRequest &mock_clone_request,
+			  librbd::MockTestImageCtx *mock_parent_imctx,
+                          int r) {
```

Comment:
Why not pass mock_parent_imctx by reference? To be consistent with other methods like expect_open/close_image.